src/index.html

```diff
@@ -6,16 +6,18 @@
     <title>Activity Tracker</title>
   </head>
   <body>
-
-    <script type="text/javascript" src="../data/users.js"></script>
-    <script type="text/javascript" src="../data/hydration.js"></script>
-    <script type="text/javascript" src="../data/sleep.js"></script>
-    <script type="text/javascript" src="../data/activity.js"></script>
-    <script type="text/javascript" src="../src/UserRepository.js"></script>
-    <script type="text/javascript" src="../src/Hydration.js"></script>
-    <script type="text/javascript" src="../src/Sleep.js"></script>
-    <script type="text/javascript" src="../src/User.js"></script>
-    <script type="text/javascript" src="../src/Activity.js"></script>
-    <script type="text/javascript" src="../src/scripts.js"></script>
+    <main>
+    </main>
   </body>
+
+  <script type="text/javascript" src="../data/users.js"></script>
+  <script type="text/javascript" src="../data/hydration.js"></script>
+  <script type="text/javascript" src="../data/sleep.js"></script>
+  <script type="text/javascript" src="../data/activity.js"></script>
+  <script type="text/javascript" src="../src/UserRepository.js"></script>
+  <script type="text/javascript" src="../src/Hydration.js"></script>
+  <script type="text/javascript" src="../src/Sleep.js"></script>
+  <script type="text/javascript" src="../src/User.js"></script>
+  <script type="text/javascript" src="../src/Activity.js"></script>
+  <script type="text/javascript" src="../src/scripts.js"></script>
 </html>
```

Comment:
I have always been told to add the scripts before the end of the body. Did you take these out of the body for any particular reason?